src/transformers/models/opt/modeling_tf_opt.py

```diff
@@ -618,7 +618,7 @@ def call(
             attention_mask = tf.ones(inputs_embeds.shape[:2], dtype=tf.bool)
         else:
             tf.debugging.assert_equal(
-                attention_mask.shape[1],
+                tf.shape(attention_mask)[1],
```

Comment:
perhaps our good old `shape_list` instead?

Reply:
Either would work here! The main thing is that we can't use the static `attention_mask.shape` because it causes compilation to fail as that dimension can be `None`.